core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/FunctionExpression.kt

```diff
@@ -1,12 +1,12 @@
-package ch.ergon.dope.resolvable.expression.unaliased.type.conditional
+package ch.ergon.dope.resolvable.expression.unaliased.type
 
 import ch.ergon.dope.DopeQuery
 import ch.ergon.dope.resolvable.expression.TypeExpression
 import ch.ergon.dope.resolvable.expression.UnaliasedExpression
 import ch.ergon.dope.resolvable.operator.FunctionOperator
 import ch.ergon.dope.validtype.ValidType
 
-sealed class ConditionalExpression<T : ValidType>(
+open class FunctionExpression<T : ValidType>(
```

Comment:
I don't like that you have to have it open... Maybe just make it abstract?

Reply:
@pgruntz What do you think?

Reply:
well, I would leave it _sealed_ and move all the subclasses into the same packages. Maybe you can create a common package "function" of the packages _relational_ and _conditional_ and put this class there

Reply:
I moved the function type expressions into a separate package with the `FunctionExpression`, which I made abstract ([because sealed classes are not usable in subpackages](https://youtrack.jetbrains.com/issue/KT-44895/Ergonomics-with-large-sealed-class-interface-hierarchies-sealed-beyond-package)).